INFO:root:train size: 20, test size: 343
INFO:root:X_test_noise shape: (343, 24)
2022-12-22 11:55:58,639	INFO worker.py:1518 -- Started a local Ray instance.
[36m(partial_opw pid=28620)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: divide by zero encountered in true_divide
[36m(partial_opw pid=28620)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28620)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 0
[36m(partial_opw pid=28620)[39m   warnings.warn('Warning: numerical errors at iteration %d' % ii)
[36m(partial_opw pid=28620)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: overflow encountered in true_divide
[36m(partial_opw pid=28620)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28622)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: divide by zero encountered in true_divide
[36m(partial_opw pid=28622)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28622)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 0
[36m(partial_opw pid=28622)[39m   warnings.warn('Warning: numerical errors at iteration %d' % ii)
[36m(partial_opw pid=28621)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: divide by zero encountered in true_divide
[36m(partial_opw pid=28621)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28621)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 0
[36m(partial_opw pid=28621)[39m   warnings.warn('Warning: numerical errors at iteration %d' % ii)
[36m(partial_opw pid=28621)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: overflow encountered in true_divide
[36m(partial_opw pid=28621)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28623)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: divide by zero encountered in true_divide
[36m(partial_opw pid=28623)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28623)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:492: UserWarning: Warning: numerical errors at iteration 0
[36m(partial_opw pid=28623)[39m   warnings.warn('Warning: numerical errors at iteration %d' % ii)
[36m(partial_opw pid=28623)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: overflow encountered in true_divide
[36m(partial_opw pid=28623)[39m   u = 1. / nx.dot(Kp, v)
[36m(partial_opw pid=28622)[39m /home/phu/.local/lib/python3.10/site-packages/ot/bregman.py:485: RuntimeWarning: overflow encountered in true_divide
[36m(partial_opw pid=28622)[39m   u = 1. / nx.dot(Kp, v)


100%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ| 6860/6860 [00:05<00:00, 1299.05it/s]
INFO:root:Accuracy: 0.7405247813411079
INFO:root:cost time: 10.808985710144043